lineman/app/components/group_page/group_theme/group_theme.haml

```diff
@@ -1,42 +1,43 @@
-.group-theme.lmo-row
-  .group-theme__cover{ng-style:'coverStyle()'}
-    .group-theme__upload-photo{ng-mouseover: 'themeHoverIn()', ng-mouseleave: 'themeHoverOut()'}
-      %button.lmo-btn-link-no-underline{ng-click: 'openUploadCoverForm()', ng-if: 'canUploadPhotos()'}
-        %i.fa.fa-lg.fa-camera
-        %span.group-theme__upload-help-text{ng-show: 'themeHover'} new photo
+.group-theme
+  .lmo-row
+    .group-theme__cover{ng-style:'coverStyle()'}
+      .group-theme__upload-photo{ng-mouseover: 'themeHoverIn()', ng-mouseleave: 'themeHoverOut()'}
+        %button.lmo-btn-link-no-underline{ng-click: 'openUploadCoverForm()', ng-if: 'canUploadPhotos()'}
```

Comment:
FYI - I'm noticing that using utility classes is ruining the nice haml. I really want us to move to using a single BEM class if possible and just mixing in the utility at the css level. 

I think it starts by making all utility classes mixins with the same name. So the haml is:
```
%button.group-theme__upload-button{...}
```
and the css is

```
.group-theme__upload-button {
   @include btnLinkNoUnderline
}
```


Does that feel ok to you? I really think we need to steer away from the utility classes now before people copy them any more. 

2 questions I have are.. 
- should we stop using all the bootstrap utilities (even the .btn- ones?).. I think so.. and we make mixins with what we want for our own buttons
- Should we namespace the mixin names like we've namespaced the utility classes.. and I think probably yes to that too.

I don't want to ban the utility classes but just only use them when it really makes no sense to mixin them and keep the BEM naming strong.